test/scroll-container.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>vaadin-horizontal-layout tests</title>
```

Comment:
nit: rename to `vaadin-scroll-container`

Reply:
done